multicluster/cmd/multicluster-controller/leader.go

```diff
@@ -63,10 +64,15 @@ func runLeader(o *Options) error {
 	if err = memberClusterStatusManager.SetupWithManager(mgr); err != nil {
 		return fmt.Errorf("error creating MemberClusterAnnounce controller: %v", err)
 	}
+
+	noCachedClient, err := client.New(mgr.GetConfig(), client.Options{Scheme: mgr.GetScheme(), Mapper: mgr.GetRESTMapper()})
```

Comment:
Leader is running in a Namespace scope, it's suppose to watch those resources in the same Namespace where the leader controller is running. I didn't see you set the Namespace here.
Do we really need to use no cached client in order to remove the `list` in the RBAC? I feel it's better to keep the same as other controllers. @jianjuns  any comment for this?

Reply:
I do not know all the implication here, but I would watch only in the same Namespace.

Another question - in RBAC configuration, we do not create a "Role" to restrict the permissions to only the “multcluster" Namespace?

Reply:
@jianjuns  we defined a 'Role' `antrea-mc-controller-role` in antrea-multicluster-leader-namespaced.yml, Antrea mc-controller will only be able to watch resources under “antrea-multicluster" Namespace.